src/FsAutoComplete.Core/Commands.fs

```diff
@@ -1156,7 +1156,7 @@ type Commands(checker: FSharpCompilerServiceChecker, state: State, hasAnalyzers:
 
 
   let fillHelpTextInTheBackground decls (pos: Position) fn getLine =
-    let declName (d: DeclarationListItem) = d.Name
+    let declName (d: DeclarationListItem) = d.NameInList
```

Comment:
Great, I do the same choice, but because error/warning message was confusing I was not sure what to do. That's probably information more important to FSharp team, but anyway.